dashboard/app/models/user.rb

```diff
@@ -858,9 +858,16 @@ def teacher?
   end
 
   def authorized_teacher?
-    # you are "really" a teacher if you are a teacher in any cohort for an ops workshop or in a plc course
-    admin? || (teacher? && (cohorts.present? || plc_enrollments.present?)) ||
-      permission?(UserPermission::LEVELBUILDER)
+    # You are an authorized teacher if you are an admin, have the AUTHORIZED_TEACHER or the
+    # LEVELBUILDER permission, or are a teacher in a cohort or PLC course.
+    return true if admin?
```

Comment:
Levelbuilders definitely should be so that they can see the teacher markdown that they are editing. Anyone have context for why admins previously were?